integration/aggregate_documents_compat_test.go

```diff
@@ -866,9 +866,6 @@ func TestAggregateCompatGroupSum(t *testing.T) {
 		Remove("ArrayInt32s").
 		Remove("Mixed").
 		Remove("ArrayAndDocuments").
```

Comment:
Current error
```
        	            	--- expected
        	            	+++ actual
        	            	@@ -8,3 +8,3 @@
        	            	     "sum": {
        	            	-      "$f": 84.13000000000011
        	            	+      "$f": 84.13000000000001
        	            	     }
        	Test:       	TestAggregateCompatGroupSum/GroupNullID/TestAggregateCompatGroupSum_Doubles
```

Still not correct, but closer now. I don't understand where `0.00000000000011` of `84.13000000000011` comes from. Because only fraction decimals in our `Doubles` collection are `double` and `double-smallest`, neither contain `0.00000000000011`. https://github.com/FerretDB/FerretDB/blob/89d65626200aab2fd5997930fb23117f8618884b/integration/shareddata/scalars.go#L130 and https://github.com/FerretDB/FerretDB/blob/89d65626200aab2fd5997930fb23117f8618884b/integration/shareddata/scalars.go#L133